scopesim_templates/metis/laser.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+"""WCU laser spectrum for METIS."""
+
+import numpy as np
+from astropy import units as u
+from astropy.io import fits
+
+from spextra import Spextrum
+
+from ..rc import Source, im_plane_utils
+
+
+def laser_spectrum(img_size=(12, 12),
+                   waves=np.arange(3, 6, .001)*u.um,
+                   centers=[3.39, 4.73, 5.263]*u.um,
```

Comment:
Also from the template manual:

> The four N band laser sources attached to the WCU during AIT are: fixed lasers at 9, 10, 11, 12 microns. They can all be shone at once.
> 
> Internal wavelength calibration with the N band laser sources will produce a spectrum with four lines.

So maybe we can create both a `laser_spectrum_n()` and a `laser_spectrum_lm()` function?

Reply:
The separate N band version wasn't mentioned in AstarVienna/METIS_Simulations#11. But I now tend towards your proposed solution of two separate functions. Both of these might call a more abstract `laser_spectrum` with the appropriate numbers (while allowing manual overriding via `kwargs`). That would also get rid of the "mutable default arguments" problem.